### DISCUSSION TITLE
Welcome to pbidea Discussions!This is a set of potentially powerful PowerBuilder function extension library, including JSON, XML, httpclient, encryption / decryption, webapi client / server, websocket client / server, FTP / SFTP, PDF view, compress / decompress, image processing, chart, map and other particularly easy-to-use functions

### DISCUSSION THREAD

**Discussion by lxb320124** [lxb320124/pbidea#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zulmach** [comment]
Hi,
Iam zulmach just an ordinary powerbuilder doers.

wondering if httprequest can send body from a file.

something like sending json formatted string but from a json file *.json
you already have incredible uo_json userobject to conver dw/ds to json file
but it would be great if uo_httpclient object can also send json body by read from a file
or is it already there ? please help pointed on this

since pb has limit and drawback on handling large string memory my pb code always crashing 
when load string from +100MB json file and try to send it to httprequest as string body
any suggestion on this ?

Thank you, all efforts are really appreciated

best regards
Zuilmach
